src/routes/impressum/+page.svelte

```diff
@@ -1,85 +1,76 @@
-	const footerMenuLoggedIn: MenuItem[] = [
-		['Start', '/'],
-		['Intern', '/backend']
-	];
 </script>
 
-<Header menu={data.loggedIn ? headerMenuLoggedIn : headerMenu} />
+<Header menu={data.loggedIn ? Menu.headerIn : Menu.headerOut} />
 
 <div class="page">
 	<div class="content-wrapper">
-		<HeadlinePage>Impressum</HeadlinePage>
-		<Spacer --height="5rem" />
+		<HeadlinePage classes="impressum-headline">Impressum</HeadlinePage>
 
 		<TextLine>Michael Gerhold</TextLine>
 		<TextLine>c/o LEADERLY UG (haftungsbeschränkt)</TextLine>
 		<TextLine>Carl-Zeiss-Str. 4</TextLine>
-		<TextLine>14727 Premnitz OT Mögelin</TextLine>
-		<SpacerOneLine />
+		<TextLine classes="impressum-one-line-margin">14727 Premnitz OT Mögelin</TextLine>
 		<TextLine>
 			E-Mail:
-			<Link classes="inline" href="mailto:info@test-conf.de" newTab={true}>info@test-conf.de</Link>
+			<Link
+				classes="inline"
+				href="mailto:info@test-conf.de"
+				newTab={true}
+				title="Klicke hier um uns eine Mail zu schreiben">info@test-conf.de</Link
+			>
 		</TextLine>
 
-		<Spacer --height="5rem" />
-		<SubHeadline>Hinweis gemäß Online-Streitbeilegungs-Verordnung</SubHeadline>
-		<Spacer --height="2rem" />
+		<SubHeadline classes="impressum-subheadline"
+			>Hinweis gemäß Online-Streitbeilegungs-Verordnung</SubHeadline
+		>
 
 		<Paragraph
 			>Nach geltendem Recht sind wir verpflichtet, Verbraucher auf die Existenz der Europäischen
 			Online-Streitbeilegungs-Plattform hinzuweisen, die für die Beilegung von Streitigkeiten
 			genutzt werden kann, ohne dass ein Gericht eingeschaltet werden muss. Für die Einrichtung der
 			Plattform ist die Europäische Kommission zuständig. Die Europäische
 			Online-Streitbeilegungs-Plattform ist hier zu finden:
-			<Link classes="inline" href="http://ec.europa.eu/odr" newTab={true}>
-				http://ec.europa.eu/odr</Link
-			>. Unsere E-Mail lautet:
-			<Link classes="inline" href="mailto:info@test-conf.de" newTab={true}>info@test-conf.de</Link>
+			<Link
+				classes="inline"
+				href="http://ec.europa.eu/odr"
+				newTab={true}
+				title="Klicke hier um dich über die Europäische
+			Online-Streitbeilegungs-Plattform zu informieren"
+			>
+				http://ec.europa.eu/odr
+			</Link>
+			. Unsere E-Mail lautet:
+			<Link
+				classes="inline impressum-one-line-margin"
+				href="mailto:info@test-conf.de"
+				newTab={true}
+				title="Klicke hier um uns eine Mail zu schreiben"
+			>
+				info@test-conf.de
+			</Link>
 		</Paragraph>
-		<SpacerOneLine />
 		<Paragraph>
 			Wir weisen aber darauf hin, dass wir nicht bereit sind, uns am Streitbeilegungsverfahren im
 			Rahmen der Europäischen Online-Streitbeilegungs-Plattform zu beteiligen. Nutzen Sie zur
 			Kontaktaufnahme bitte unsere obige E-Mail und Telefonnummer.
 		</Paragraph>
 
-		<Spacer --height="5rem" />
-		<SubHeadline>Disclaimer – rechtliche Hinweise</SubHeadline>
-		<Spacer --height="2rem" />
+		<SubHeadline classes="imperssum-subheadline">Disclaimer – rechtliche Hinweise</SubHeadline>
```

Comment:
```suggestion
		<SubHeadline classes="impressum-subheadline">Disclaimer – rechtliche Hinweise</SubHeadline>
```

or rather

`imprint-subheadline`